code/modules/mob/mob.dm

```diff
@@ -917,8 +917,12 @@ var/list/slot_equipment_priority = list( \
 	if(ghost)
 		ghost.notify_cloning(message, sound)
 		return ghost
-
-
+		
+/mob/proc/force_reentry()
+	var/mob/dead/observer/ghost = get_ghost()
+	if(ghost)
+		ghost.reenter_corpse()
+		return ghost
```

Comment:
Not sure as to the purpose of returning the observer mob object if they've re-enetered? A better way would to be to have it return either 1, or 0 (success and fail/true and false respectively) if you need to detect whether or not the procedure has completed/failed.